src/compositionFunctions/useLocalStorage.js

```diff
@@ -0,0 +1,36 @@
+import { LocalStorage } from 'quasar'
+
+export default function () {
+  const LOCAL_STORAGE_KEY = {
```

Comment:
move outside of functions since this is a const and does not to reinitialize every time